verifier.go

```diff
@@ -131,6 +133,30 @@ func (v *Verifier) EnableSMTPCheck() *Verifier {
 	return v
 }
 
+// EnableGmailCheckByAPI Gmail API verifier is activated when EnableSMTPCheck.
+// If client is nil, will use http.DefaultClient.
+// Certain API verifiers require additional input, for instance, Hotmail necessitates the use of a WebDriver.
+// Gmail API verifier only need a http client.
+func (v *Verifier) EnableGmailCheckByAPI(client *http.Client) *Verifier {
+	v.apiVerifiers["gmail"] = newGmailAPIVerifier(client)
+	return v
+}
+
+// EnableYahooCheckByAPI Yahoo API verifier is activated when EnableSMTPCheck.
+// If client is nil, will use http.DefaultClient
+func (v *Verifier) EnableYahooCheckByAPI(client *http.Client) *Verifier {
+	v.apiVerifiers["yahoo"] = newYahooAPIVerifier(client)
+	return v
+}
+
+func (v *Verifier) DisableGmailCheckByAPI() {
+	delete(v.apiVerifiers, "gmail")
+}
+
+func (v *Verifier) DisableYahooCheckByAPI() {
+	delete(v.apiVerifiers, "yahoo")
+}
+
```

Comment:
It's a bit wordy to create a functioning pair for every vendor. Can change to something like `Enable/DisableAPIVerifer(name string)`.

Reply:
Hi. @git-hulk 
I have left a comment before.
> // Certain API verifiers require additional input, for instance, Hotmail necessitates the use of a WebDriver.

If using the way you mentioned above, the function parameter will expand as time goes on. Thus i thought every APIVerifier should have it's own custom initialize function and params.

Please refer to https://github.com/reacherhq/check-if-email-exists/blob/master/core/src/smtp/microsoft/hotmail.rs#LL57C2-L57C11

I think we will extend new verifiers later.

Reply:
Got it, thanks.